impl11/ddraw/EffectsRenderer.cpp

```diff
@@ -981,7 +981,7 @@ void EffectsRenderer::ApplyDiegeticCockpit()
 		// throttle are present.
 		//if (!_bThrottleTransformReady)
 		{
-			float throttle = (DiegeticMesh == DM_THR_ROT_X || DM_THR_ROT_Y || DM_THR_ROT_Z) ?
+			float throttle = (DiegeticMesh == DM_THR_ROT_X || DiegeticMesh == DM_THR_ROT_Y || DM_THR_ROT_Z) ?
```

Comment:
I tried to fix a (potential) bug I detected thanks to a VS warning but did it only half-way.
In the original code, the conditional expression on the right of of the assignment will always be true because DM_THR_ROT_Y and DM_THR_ROT_Z are not zero.

If you want to do "DiegeticMesh is one of (DM_THR_ROT_X, DM_THR_ROT_Y, DM_THR_ROT_Z), you need to compare with each one explicitly.

If the constants are a bit mask, then you should use the bitwise OR ("|"), but I don't think it's the case.